packages/atlas/src/components/_icons/JoyTokenIcon/JoyTokenIcon.tsx

```diff
@@ -25,6 +25,7 @@ export type JoyTokenIconProps = {
   variant?: JoyTokenIconVariant
   size?: JoyTokenIconSize
   className?: string
+  hasShadow?: boolean
```

Comment:
Like mentioned in rocket chat, let's remove this prop and use monochrome version instead, just with a different color applied